windows/src/desktop/kmshell/util/Keyman.System.KeymanStartTask.pas

```diff
@@ -12,7 +12,11 @@   TKeymanStartTask = class
   private
     class function GetTaskName: string;
     class procedure CleanupAlphaTasks(pTaskFolder: ITaskFolder); static;
+  {$IF FALSE}
```

Comment:
Just stopping a compiler warning, no functional change to this unit

Reply:
... so this disables warnings, but doesn't undefine the method?

Reply:
It does undefine the method, because it isn't currently used. But that's only because we've disabled this functionality for now. Once (if) we re-enable, we'll add a call to `DeleteTask` somewhere, remove these changes here, and the warning would go away again.

Reply:
Feels like an awkward throw-in (the warning wasn't caused by this PR's changes), but... okay, I guess.